core/src/main/scala/org/apache/spark/ui/exec/ExecutorsTab.scala

```diff
@@ -62,6 +64,7 @@ class ExecutorsListener(storageStatusListener: StorageStatusListener) extends Sp
   override def onExecutorAdded(executorAdded: SparkListenerExecutorAdded): Unit = synchronized {
     val eid = executorAdded.executorId
     executorToLogUrls(eid) = executorAdded.executorInfo.logUrlMap
+    executorToTotalCores(eid) = executorAdded.executorInfo.totalCores
```

Comment:
so this might have issues on other cluster manager other then yarn.. see the discussion at https://github.com/apache/spark/pull/2980